src/Documenter.jl

```diff
@@ -638,18 +651,17 @@ julia> Documenter.generate("MyPackageName")
 ```
 """
 function generate(pkgname::AbstractString; dir=nothing)
-    # TODO:
-    #   - set up deployment to `gh-pages`
```

Comment:
I tried my darnedest to figure out how to make deploy keys with the github api to no avail; more info here: https://developer.github.com/v3/repos/keys/ .